src/player.cpp

```diff
@@ -1819,6 +1819,10 @@ int player::run_cost( int base_cost, bool diag ) const
         // Rationale: Average running speed is 2x walking speed. (NOT sprinting)
         stamina_modifier *= 2.0;
     }
+    if( move_mode == "crouch" )
+    {
+        stamina_modifier *= 0.5;
```

Comment:
I don't know if this makes sense. You are making effort, after all. Maybe `stamina_modifier` should be 1.25 instead.

Reply:
`stamina_modifier` is just a speed modifier, not stamina (idk why it's like that). I've made is so crouching decreases speed 0.5 times, not making it easier to crouch than walking. Besides, crouching is not "making effort", I mean, not **that** hard if you are moving with your comfortable pace.